### PR TITLE
Fetch api data

### PR DESCRIPTION
**Pull request by lorenzoworx**
In this pull request, I've implemented the following changes
- Use the following external API: [API Ninjas](https://api-ninjas.com/api/quotes)
- Create a new component for displaying a quote
- Fetch data inside of a new component
- Show a `loading` state
- Show an `error` state
- Import the new component into your main component (next to the Calculator)

### CODE REVIEW

File: src/components/Quotes.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const Quotes = () => {
+  const [quote, setQuote] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness',
+        {
+          headers: { 'X-Api-Key': 'VF/Pc9XBd+WWEkCjPg18aA==RDypQjoE00TFNCQq' },
```
Comment (Ismailco) [inline src/components/Quotes.jsx]: - [ ] [OPTIONAL] Kindly make sure to not expose any API keys in your code, you can use environment variables to add the API key securely to your code

**Review by Ismailco — APPROVED** [review]
### Hi @lorenzoworx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ Uses API correctly
✔️ Handles Error & Loading state
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Quotes.jsx (hunk 1 above)
Comment (lorenzoworx) [reply src/components/Quotes.jsx:12]: Thanks for the tip. Can you please walk me through implementing this or maybe point me in the direction of a relevant tutorial

**Comment by Ismailco** [comment]
@lorenzoworx, sorry for the late response, I see that you found the correct way to use the environment variable.
here is a youtube video if you want to check it out anyway 👉🏼 https://www.youtube.com/watch?v=7oJgdyMS4rQ

**Pull request merged**